packages/fxa-content-server/server/lib/version.js

```diff
@@ -78,7 +78,8 @@ function getL10nVersion() {
       __dirname,
       '..',
       '..',
-      'fxa-content-server-l10n',
```

Comment:
Not sure if it's relevant, but here's the other (non-markdown/changelog) instances of `fxa-content-server-l10n` in the codebase:

```sh
git grep "fxa-content-server-l10n" ':!*.md' | cat

.dockerignore:**/fxa-content-server-l10n
.github/workflows/l10n-gettext-extract.yml:          repository: mozilla/fxa-content-server-l10n
.gitignore:packages/fxa-auth-server/fxa-content-server-l10n
.gitignore:packages/fxa-content-server/fxa-content-server-l10n/
.gitignore:packages/fxa-payments-server/fxa-content-server-l10n/
.gitignore:packages/fxa-react/fxa-content-server-l10n/
.gitignore:packages/fxa-settings/fxa-content-server-l10n/
_scripts/l10n/clone.sh:FXA_L10N_REPO="${FXA_L10N_REPO:-https://github.com/mozilla/fxa-content-server-l10n.git}"
packages/fxa-auth-server/.prettierignore:fxa-content-server-l10n/*
# packages/fxa-auth-server/scripts/e2e-email/validate-email.js:      // See https://github.com/mozilla/fxa-content-server-l10n/issues/44 about sr-LATN
packages/fxa-content-server/.prettierignore:fxa-content-server-l10n/*
packages/fxa-content-server/server/bin/fxa-content-server.js:logger.info(`fxa-content-server-l10n commit hash set to: ${version.l10n}`);
packages/fxa-payments-server/.eslintignore:fxa-content-server-l10n
# packages/fxa-payments-server/Gruntfile.js:      // We need this for tests because we pull the latest from `fxa-content-server-l10n`
# packages/fxa-react/Gruntfile.js:      // We need this for tests because we pull the latest from `fxa-content-server-l10n`
# packages/fxa-settings/Gruntfile.js:      // We need this for tests because we pull the latest from `fxa-content-server-l10n`
```

Reply:
I will file a separate PR to fix some of these references. A few are valid since they point at the l10n repo. The others are just comments. As far as I can tell, none of them appear to be an actual problem though.